intel_extension_for_transformers/backends/neural_engine/graph/scripts/convert_chatglm.py

```diff
@@ -0,0 +1,382 @@
+#  Copyright (c) 2023 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import argparse
+import platform
+import struct
+import sys
+from enum import Enum
+from pathlib import Path
+
+import torch
+from tabulate import tabulate
+from tqdm import tqdm
+from transformers import AutoModel, AutoTokenizer
+
+NE_QK8_0 = 32
+NE_QK4_0 = 32
+NE_QK4_1 = 32
+NE_QK5_0 = 32
+NE_QK5_1 = 32
+
+NE_MEM_ALIGN = 16
+
+if platform.system() == "Darwin":
+    # cpm_kernels doesn't support macOS but transformers will check missing packages, so mock it
+    sys.modules["cpm_kernels"] = object()
+
+
+class NEType(Enum):
+    F32 = 0
+    F16 = 1
+    Q4_0 = 2
+    Q4_1 = 3
+    Q5_0 = 6
+    Q5_1 = 7
+    Q8_0 = 8
+
+
+class ModelType(Enum):
+    CHATGLM = 1
+    CHATGLM2 = 2
+
+
+def quantize_q8_0(tensor: torch.Tensor) -> torch.CharTensor:
+    # equivalent to ne_quantize_q8_0 in ne.c
```

Comment:
we don't need quantization in scripts

Reply:
https://github.com/intel/intel-extension-for-transformers/pull/107 provides a quant cpp file.